doc/Makefile.in

```diff
@@ -122,19 +122,19 @@ XML_SOURCES = \
 	performance_tips.xml \
 	postgis.xml \
 	reference_accessor.xml \
+	reference_bbox.xml \
```

Comment:
ST_Envelope, ST_OrientedEnvelope I'd expect nearby. Naming needs tweaking then - how about calling section with something like "rectangle functions" so box / geometry-with-rect-angles distinction is not there?

This may also consume operator section since these are defined on boxes?

Reply:
See previous comment.  The Bounding Box section does not contain either "Envelope" function, so I think this is ok?  When I reorg the Accessors and Processing sections we can think about where they should go.

My opinion is that they belong in different sections, even though they are unfortunately named similarly.  One is just a simple accessor, but the OrientedEnvelope requires a full-blown spatial algorithm to compute.  I think it should be called OrientedRectangle or something like that.  It should not mention Envelope, because that implies an axis-oriented rectangle.

Reply:
I think the operator section should remain distinct, since it's long already.  And operators are clearly different things to functions.